x-pack/test/api_integration/apis/metrics_ui/inventory_threshold_alert.ts

```diff
@@ -19,7 +19,7 @@ import { InventoryItemType } from '../../../../plugins/infra/common/inventory_mo
 
 export default function ({ getService }: FtrProviderContext) {
   const esArchiver = getService('esArchiver');
-  const esClient = convertToKibanaClient(getService('es'));
+  const esClient = getService('es');
```

Comment:
Was this the cause of the failure? Interesting...

Reply:
Yup, not sure why but that fixed it.